scss/_patterns_tabs.scss

```diff
@@ -71,20 +66,88 @@
         }
       }
 
+      &:active,
+      &[aria-selected='true'] {
+        // Hide the highlight when focussing (in combination with the parent
+        // states) in browsers that don't support focus-visible.
+        &:focus::before,
+        &:focus::after {
+          content: none;
+        }
+      }
+    }
+  }
+
+  // Theming
+  @if ($theme-default-p-tabs == 'dark') {
+    .p-tabs {
+      @include vf-p-tabs-dark-theme;
+    }
+
+    .p-tabs.is-light {
+      @include vf-p-tabs-light-theme;
+    }
+  } @else {
+    .p-tabs {
+      @include vf-p-tabs-light-theme;
+    }
+
+    .p-tabs.is-dark {
+      @include vf-p-tabs-dark-theme;
+    }
+  }
+}
+
+@mixin vf-p-tabs-theme($colors--text-default, $colors--border-default, $color-highlight-bar-hover) {
```

Comment:
I guess addition of the themes made CSS increase in size above the test threshold, so we need to bump the threshold again… (to make tests pass).